### PR TITLE
change quickstart download url

### PR DESCRIPTION
**Pull request by zhoujinsong**

### CODE REVIEW

**Comment by hameizi** [comment]
LGTM

File: site/docs/getting-started.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ tar -zxvf flink-1.12.7-bin-scala_2.12.tgz
 # 下载 hadoop 依赖
```
Comment (zstraw) [inline site/docs/getting-started.md:72]: line:55 can be removed

File: site/docs/getting-started.md [hunk 2]
```diff
@@ -72,7 +72,7 @@ tar -zxvf flink-1.12.7-bin-scala_2.12.tgz
...
 wget https://repo1.maven.org/maven2/org/apache/flink/flink-shaded-hadoop-2-uber/${HADOOP_VERSION}-10.0/flink-shaded-hadoop-2-uber-${HADOOP_VERSION}-10.0.jar
 # 下载 arctic flink connector
-wget XXX
+wget https://github.com/NetEase/arctic/releases/download/v0.3.0-rc1/arctic-flink-runtime-1.12-0.3.0.jar
```
Comment (zstraw) [inline site/docs/getting-started.md:75]: Shall we make flink 1.12 alternative? 1.14 would be fine.

File: site/docs/getting-started.md (hunk 1 above)
Comment (zhoujinsong) [reply site/docs/getting-started.md:72]: yes

File: site/docs/getting-started.md (hunk 2 above)
Comment (zhoujinsong) [reply site/docs/getting-started.md:75]: I prefer use specific flink version in quickstart and leave supported flink version description in flink chapter

**Pull request merged**